src/pages/posting/components/NewPost/NewPost.tsx

```diff
@@ -78,22 +80,30 @@ const NewPost = ({ meditationInfo, customToken }: NewPostProps) => {
 
   return (
     <>
+      {posting.length >= 500 && (
+        <Toast
+          content='포스팅은 최대 500글자까지 가능합니다.'
+          type='WARNING'
+        />
+      )}
```

Comment:
```suggestion
      {posting.length > 501 && (
        <Toast
          content='포스팅은 최대 500글자까지 가능합니다.'
          type='WARNING'
        />
      )}
```
지금은 501자까지 작성가능합니다! >= 를 > 로 바꿔야 할 것 같아요!
그리고 500숫자는 상수화하면 좋겠네요!

Reply:
textarea 에 maxLength 를 설정해 주어서 500자를 초과하여 글을 작성할 수 없어요...!! 그래서 500자로 해놓은 거랍니다! 숫자는 상수화처리 해놓을게요!

Reply:
엇 제 테스트 환경에서는 500자에서 토스트 UI가 뜨고 501자까지 써지는 것 같던데요...!

Reply:
아하 그리고 리뷰 내용은 제가 잘못봤군요!ㅋㅋㅋ

Reply:
확인해보겠습니다!!

Reply:
아.. 요게 또 한글의 경우 모음이 한자 더 써지는 문제가 있네요.. ㅋㅋㅋㅋ ㅠㅠㅠ
그런데 영어의 경우는 딱 500자 까지만 써집니다 ㅠㅠ 영어를 생각하면 500자 이상으로 둬도 될까요?


<img width="75" alt="image" src="https://github.com/prgrms-fe-devcourse/FEDC4_NIRVANA_Gidong/assets/80307321/f2f5f628-2ebc-4335-aba7-79047b08f45e">
<img width="95" alt="image" src="https://github.com/prgrms-fe-devcourse/FEDC4_NIRVANA_Gidong/assets/80307321/dc82c740-f884-446e-b4f3-638346f66374">